dotnet-test-samples/kinesis-lambda-dynamodb/tests/KinesisEventHandler.UnitTests/Functions/ProcessEmployeeFunctionTests.cs

```diff
@@ -0,0 +1,80 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Amazon.Lambda.TestUtilities;
+using FluentAssertions;
+using KinesisEventHandler.Functions;
+using KinesisEventHandler.Repositories;
+using KinesisEventHandler.Repositories.Mappers;
+using KinesisEventHandler.Repositories.Models;
+using KinesisEventHandler.UnitTests.Utilities;
+using Moq;
+using Xunit;
+
+namespace KinesisEventHandler.UnitTests.Functions;
+
+public class ProcessEmployeeFunctionTests
+{
+    [Fact]
+    public Task ProcessEmployeeFunction_Should_ExecuteSuccessfully()
+    {
+        //Arrange
+        var repository = new Mock<IDynamoDbRepository<EmployeeDto>>();
+
+        repository.Setup(x =>
+                x.PutItemAsync(It.IsAny<EmployeeDto>(), It.IsAny<CancellationToken>()))
+            .ReturnsAsync(UpsertResult.Inserted);
+
+        var sut = new ProcessEmployeeFunction(repository.Object);
+        var employee = new EmployeeBuilder().Build();
+        var context = new TestLambdaContext();
+
+        //Act
+        var taskResult = sut.ProcessKinesisRecord(employee, context);
+
+        //Assert
+        Assert.True(taskResult.IsCompleted);
+        return Task.CompletedTask;
```

Comment:
That will always be the case, the assert checks that C#/.NET works :)
Instead assume that if you got this far no exception was thrown and instead ask what was the result that the user cares about (requirement) - for example a record was saved in the DB.